breathecode/jobs/tests/actions/tests_fetch_data_to_json.py

```diff
@@ -0,0 +1,233 @@
+    '_type':
+    'dict'
+}, {
+    'Searched_job': 'junior web developer',
+    'Job_title': 'Pentester Cybersecurity',
+    'Location': 'Remote (Chile)',
+    'Company_name': 'Rule 1 Ventures',
+    'Post_date': 'November 05, 2021',
+    'Extract_date': '2022-01-30',
+    'Job_description': 'Vuln exploitation Security reports',
+    'Salary': 'Not supplied',
+    'Tags': ['back-end', 'cybersecurity', 'english', 'pentesting', 'python'],
+    'Apply_to': 'https://www.getonbrd.com/jobs/cybersecurity/security-engineer-rule-1-ventures-remote',
+    '_type': 'dict'
+}, {
+    'Searched_job': 'junior web developer',
+    'Job_title': 'Pentester Cybersecurity',
+    'Location': 'Remote (Chile, Venezuela)',
+    'Company_name': 'Rule 1 Ventures',
+    'Post_date': 'November 05, 2021',
+    'Extract_date': '2022-01-30',
+    'Job_description': 'Vuln exploitation Security reports',
+    'Salary': 'Not supplied',
+    'Tags': ['back-end', 'cybersecurity', 'english', 'pentesting', 'python'],
+    'Apply_to': 'https://www.getonbrd.com/jobs/cybersecurity/security-engineer-rule-1-ventures-remote',
+    '_type': 'dict'
+}, {
+    'Searched_job': 'junior web developer',
+    'Job_title': 'Front-end Developer',
+    'Location': 'Lima',
+    'Company_name': 'ID Business Intelligence',
+    'Post_date': 'January 24, 2022',
+    'Extract_date': '2022-01-30',
+    'Job_description':
+    'Manejo de Git Flow. (~°-°)~ Dominar a profundidad CSS y JS (mínimo 1 año) Experiencia con React Experiencia consumiendo Web Service (Rest) Preocuparse por entregar productos de calidad.',
+    'Salary': 'Not supplied',
+    'Tags': ['api', 'css', 'front-end', 'git', 'javascript', 'react'],
+    'Apply_to': 'https://www.getonbrd.com/jobs/programming/fronted-developer-id-business-intelligence-remote',
+    '_type': 'dict'
+}, {
+    'Searched_job':
+    'junior web developer',
+    'Job_title':
+    'Junior Web Developer',
+    'Location':
+    None,
+    'Company_name':
+    'Reign',
+    'Post_date':
+    'January 29, 2022',
+    'Extract_date':
+    '2022-01-30',
+    'Job_description':
+    '',
+    'Salary':
+    'Not supplied',
+    'Tags': [
+        'angularjs', 'api', 'back-end', 'ci/cd', 'css', 'docker', 'front-end', 'html5', 'javascript', 'json',
+        'mongodb', 'node.js', 'nosql', 'postgresql', 'react', 'responsive', 'ui design', 'virtualization'
+    ],
+    'Apply_to':
+    'https://www.getonbrd.com/jobs/programming/junior-web-developer-reign-remote',
+    '_type':
+    'dict'
+}]
+
+
+class ActionFetchDataToJsonTestCase(JobsTestCase):
+    """Tests action certificate_screenshot"""
+    def test_fetch_data_to_json__without_spider(self):
+        """Test /run_spider without spider"""
+        try:
+            fetch_data_to_json(None, DATA)
+        except Exception as e:
+            self.assertEquals(str(e), ('without-spider'))
+
+    def test_fetch_data_to_json__without_data(self):
```

Comment:
Mock `Logger`, `requests.get` and all the functions are used inside this fuction